bundles/org.openhab.binding.modbus/README.md

```diff
@@ -118,13 +118,14 @@ Basic parameters
 Advanced parameters
 
 | Parameter                       | Required | Type    | Default if omitted | Description                                                                                                                                                        |
-| ------------------------------- | -------- | ------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|---------------------------------| -------- | ------- |--------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `timeBetweenTransactionsMillis` |          | integer | `60`               | How long to delay we must have at minimum between two consecutive MODBUS transactions. In milliseconds.                                                            |
 | `timeBetweenReconnectMillis`    |          | integer | `0`                | How long to wait to before trying to establish a new connection after the previous one has been disconnected. In milliseconds.                                     |
 | `connectMaxTries`               |          | integer | `1`                | How many times we try to establish the connection. Should be at least 1.                                                                                           |
+| `afterConnectionDelayMillis`    |          | integer | `0`                | Delay between connection and first modbus operation .                                                                                                              |
```

Comment:
Please specify units (millis) in documentation. In addition, good to use same terminology as in UI, so one can correlate between parameter name and UI using documentation.

```suggestion
| `afterConnectionDelayMillis`    |          | integer | `0`                | Connection warm-up time, i.e. delay between connection and first modbus operation. In milliseconds.                                                                                                              |
```

(you probably need to reformat whole table as welll)